shardmanager.go

```diff
@@ -202,16 +228,25 @@ func (n *ShardManager) SubscribeTopic(
 			handler(n.ctx, msg)
 		}
 	}()
+	// Tag topic to subscribe to if nothing goes wrong
+	logger.SetTag(spanctx, "Topic", topic)
 	return nil
 }
 
-func (n *ShardManager) UnsubscribeTopic(topic string) error {
```

Comment:
I just found that only `nil` are returned in this function. Does that make sense to make this function not return anything?